server/pull_request.go

```diff
@@ -266,6 +266,57 @@ func isSpinmintDoneComment(message string, upgrade bool) bool {
 	pattern := regexp.MustCompile(spinmintDoneMessage)
 	return pattern.MatchString(message)
 }
+
+func CheckPRActivity() {
+	mlog.Info("Checking if need to Stale a Pull request")
+	var prs []*model.PullRequest
+	if result := <-Srv.Store.PullRequest().ListOpen(); result.Err != nil {
```

Comment:
Nit: You can drop the else and be compliant with Idiomatic Error Handling.

Reply:
i think is done.